presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -118,6 +118,12 @@ public InMemoryHashAggregationBuilder(
     @Override
     public void close()
```

Comment:
what about other `HashAggregationBuilder` implementation: `SpillableHashAggregationBuilder`. I think is should also free memory on `close()`.